### DISCUSSION TITLE
Dynamic landscape for a habitat quality map

### DISCUSSION THREAD

**Discussion by anyperez** [RangeShifter/RangeshiftR-tutorials#12] [Q&A] [answered]
Hi everyone,

I have a question similar to the previous one asked by David. In my case, I'm doing a stage-structured model with a dynamic landscape using a habitat quality map. I noticed that the dynamic lanscape (i.e. changing the habitat suitability for different timesteps) only works if I incorporate density dependence in the demography module (right?), and I guess this is due to the way in which habitat suitability is calculated. However, I'd like to know if, once I have reduced the habitat quality for a given timestep, is it possible to implement it so that it only affects the survival of one of the stages of the population? or the changes in habitat suitability necessarily affect the whole population in the cells where the change was made?

Thank you for your help.

**Comment by damariszurell** [comment, marked as answer]
Hi Any,

Depending on your settings, the LandscapeFile (the habitat) will affect carrying capacity (in simple demographic model) or density dependence (in stage-structured model) or it can act on dispersal costs (via Costs and StepMort in SMS, or via StepMort in CorrRW). 
You can define whether density dependence is acting on development (of all stages), survival (of all stages) or fecundity (of all stages). Additionally, you can define weights for how much each stage contributes to density. 

Ciao,
Damaris
